packages/algod_client/src/core/fetch-http-request.ts

```diff
@@ -101,15 +91,15 @@ export class FetchHttpRequest extends BaseHttpRequest {
 
     const status = extractStatus(error)
     if (status !== undefined) {
-      const retryStatuses = this.config.retryStatusCodes ?? RETRY_STATUS_CODES
+      const retryStatuses = RETRY_STATUS_CODES
       if (retryStatuses.includes(status)) {
```

Comment:
nitpick: maybe we can do this
```suggestion
      if (RETRY_STATUS_CODES.includes(status)) {
```